### DISCUSSION TITLE
Is it possible to have default `dateTimeFormats`?

### DISCUSSION THREAD

**Discussion by isgj** [intlify/vue-i18n#564] [Q&A]
When a `dateTimeFormat` is not found for the active locale it will search in the fallback locale. In this case it will show the date in the fallback locale format. Is it possible to use only the options from fallback but use the current locale. 
Ex.
```js
dateTimeFormat: {
    en: {
        short: {
            year: 'numeric',
            month: '2-digit',
            day: '2-digit'
        }
    }
}
```
If the current locale is `en-GB`, writing `$d('short', date)` will give a date like `mm/dd/yyyy`. Is it possible to use only the options in the fallback and the selected locale `en-GB` ~~(of curse without passing everywhere the current locale to `$d('short', date, currentLocale)`)~~?

**Comment by tbnobody** [comment]
I know this message is quite old, but did you found a solution?

**Reply by isgj** [reply]
not as a fallbak but you can copy the same config for all the locales

```ts
const timeFormats = {
    date: { year: 'numeric', month: '2-digit', day: '2-digit' },
    dateTime: {
        year: 'numeric',
        month: '2-digit',
        day: '2-digit',
        hour: '2-digit',
        minute: '2-digit',
    },
    time: { hour: '2-digit', minute: '2-digit' },
} as const;

// used it when creating vue-i18n
createI18n({
    // .....
    datetimeFormats: {
        en: timeFormats,
        de: timeFormats,
        nl: timeFormats,
    },
})
```
